docs/system-overview.rst

```diff
@@ -0,0 +1,79 @@
+Exam System Overview
+===========================
+This document outlines the components involved in the edX exam system. It should
+serve as a guide to orient developers on what components exist and how they interact.
+
+.. contents::
+
+System Components
+-----------------
+
+Interface with edX Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/components-internal.png
+
+Interface with Proctoring Tools
```

Comment:
Can LTI launches be done using a piece of software installed on the learner's machine? I thought LTI was all browser based.

Reply:
yeah just indirectly, once you launch to the providers webpage it can download and install software or a browser plugin (whatever really). Kinda the same thing the software download url for proctortrack does today.